libraries/libgambc/0001-Backport-macros-WHEN-and-UNLESS.patch

```diff
@@ -0,0 +1,45 @@
+From 12bae91247442210713b934276518dbba9f02459 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?J=C3=B6rg=20F=2E=20Wittenberger?= <Joerg.Wittenberger@softeyes.net>
+Date: Mon, 28 Sep 2020 12:57:22 +0200
+Subject: [PATCH] Backport macros WHEN and UNLESS
+
+---
+ lib/_nonstd.scm | 22 ++++++++++++++++++++++
+ 1 file changed, 22 insertions(+)
+
+diff --git a/lib/_nonstd.scm b/lib/_nonstd.scm
+index f1db855..e66aba4 100644
+--- a/lib/_nonstd.scm
```

Comment:
The way we include patches everything above this line (or the one above it) needs to go. See [libraries/libgambc/os_shell.c.patch](https://github.com/part-cw/lambdanative/blob/master/libraries/libgambc/os_shell.c.patch) for an example.